tox.ini

```
@@ -20,6 +20,7 @@ setenv =
     dev: ALEMBIC_CONFIG = {env:ALEMBIC_CONFIG:conf/alembic.ini}
     dev: DATABASE_URL = {env:DATABASE_URL:postgresql://postgres@localhost:5433/postgres}
     tests: DATABASE_URL = {env:UNITTESTS_DATABASE_URL:postgresql://postgres@localhost:5433/lms_tests}
+    tests: PYTHONDEVMODE = {env:PYTHONDEVMODE:1}
```

I guess we could enable it in dev and functests as well?

```suggestion
    dev,tests,functests: PYTHONDEVMODE = {env:PYTHONDEVMODE:1}
```

---

Oh, and of course this will have to be backported into the cookiecutter's repo